openssl-sys/src/handwritten/evp.rs

```diff
@@ -364,6 +364,29 @@ extern "C" {
     #[cfg(all(any(ossl111, libressl291), not(osslconf = "OPENSSL_NO_SM4")))]
     pub fn EVP_sm4_ctr() -> *const EVP_CIPHER;
 
+    #[cfg(ossl111)]
```

Comment:
As far as I can tell, these functions have existed since OpenSSL 0.9.8, why are you restricting the to OpenSSL 1.1.1+? These also appear to be in all versions of libressl. I'm also slightly confused by the boringssl annotations (it appears to not include any of these?) but I'm much less familiar with boringssl.

Reply:
To be honest I was trying to get the CI green even with restrictive attributes and then slowly relaxing them (I don't see a way of checking easily which version of OpenSSL introduced which function... The man pages on openssl home page are a little bit scarce).

Thanks for your help!

Reply:
Yeah it's a pain, the best way I've found is looking at what commit introduced them to the header files.

Reply:
Good point, git blame'ing and then following to release tag should be quite easy. For these particular ciphers it seems they were introduced long ago enough that all tested versions either contain it or filter them out with `OPENSSL_NO_xxx` defines.

I've updated the `cfg`s to be as broad as possible.

Thanks for help!